Formula/libplacebo.rb

```diff
@@ -28,24 +46,12 @@ class Libplacebo < Formula
   depends_on "sdl2"
   depends_on "vulkan-loader"
 
-  fails_with gcc: "5"
-
-  resource "Mako" do
-    url "https://files.pythonhosted.org/packages/ad/dd/34201dae727bb183ca14fd8417e61f936fa068d6f503991f09ee3cac6697/Mako-1.2.1.tar.gz"
-    sha256 "f054a5ff4743492f1aa9ecc47172cb33b42b9d993cffcc146c9de17e717b0307"
-  end
-
-  resource "MarkupSafe" do
-    url "https://files.pythonhosted.org/packages/1d/97/2288fe498044284f39ab8950703e88abbac2abbdf65524d576157af70556/MarkupSafe-2.1.1.tar.gz"
-    sha256 "7f91197cc9e48f989d12e4e6fbc46495c446636dfc81b9ccf50bb0ec74b91d4b"
-  end
-
   def install
-    python = "python3.10"
-    venv_root = buildpath/"venv"
-    venv = virtualenv_create(venv_root, python)
-    venv.pip_install resources
-    ENV.prepend_path "PYTHONPATH", venv_root/Language::Python.site_packages(python)
+    if build.stable?
```

Comment:
Why not otherwise?

Reply:
When checked out from git, these are submodules.

Reply:
You don't need the `if build.stable?` conditional, since `resources` is empty for `head`.